### ISSUE TITLE
Output from BayesDeBulk cannot be compared to other output

### ISSUE THREAD

**Issue by sgosline** [sgosline/proteomicsTumorDeconv#133]
The framework expects as output a matrix in which the rows are cell type names (same as the input signature matrix ) and the columns are patient samples (same as signature matrix). Our correlation tests currently search for overlap between row names and column names, so will not compute any correlations if there are no matches.

**Comment by annapamma** [comment]
@sgosline Just to clarify, should I include BayesDeBulk in `run-deconv.cwl` and make sure the output aligns with the others?

I know you went through `run-deconv.cwl` at the last meeting. Could I get a README or a walk-through on the command to run this? 

Thank you!

**Comment by sgosline** [comment]
Yes, assigning to myself to create a readme for the `run-deconv.cwl` tool, as it is a big one.

**Comment by sgosline** [comment]
I added the readme, but the output of the bayesdebulk tool is missing row names (corresponding to to the column names of the signature matrix) and column names (corresponding to the column names of the matrix to be decolved).
